docs/sources/mimir/configure/configuration-parameters/index.md

```diff
@@ -3368,6 +3368,19 @@ The `limits` block configures default and per-tenant limits imposed by component
 # CLI flag: -ingester.active-series-custom-trackers
 [active_series_custom_trackers: <map of tracker name (string) to matcher (string)> | default = ]
 
+# (advanced) Additional custom trackers for active metrics merged on top of the
+# custom trackers. This configuration option can be useful to define the base
```

Comment:
Does `merged on top of the custom trackers` refer to preexisting trackers? If so, we should specify this for clarity.

Reply:
Reading the rest of this, it looks like that are merged on top of base custom trackers? I think we should say that.

Reply:
Yes your understanding is correct. I've addressed it in https://github.com/grafana/mimir/pull/10428/commits/b66faea0d175b7d3cbe72bb40f014b00b52df972